docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,72 @@
+# Why Toolpad?
+
+<p class="description">Explains our motivation for building Toolpad, where it differs from other tools and how it could be useful to you.</p>
+
+## Introduction
+
+Toolpad is primarily designed as an application framework for internal tool development. Such applications are often built by teams with access to fewer frontend development resources. These teams face challenges such as:
+
+- Having to keep up with the fast evolving landscape of libraries, frameworks and best practices.
+- Lots of glue code is required to get backend data to the browser and hooked up to data displaying components.
+- As backends evolve, the internal applications need to evolve with them with minimal amount of maintenance required.
+- Lesser focus on quality assurance usually leads to higher chances of failure.
```

Comment:
these don't feel like "challenges" in the same vein as the last two list items—feels like they're just restating the same core thesis here, which is that teams often don't have the resources to maintain the front end. Maybe this list is not necessary at all?